### PR TITLE
dQdV features 

### PR DESCRIPTION
**Pull request by BruisVanVlijmen2020**
Three main changes:

1) Removing the sigma feature - the lmfit package still doesn't work. 
2) Adding cwt_range as a parameter in isolate_dQdV_peaks() function, to improve robustness 'find finder function'
3) Adding through feature for dQdV

### CODE REVIEW

File: beep/features/featurizer_helpers.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def isolate_dQdV_peaks(processed_cycler_run, diag_nr, charge_y_n, max_nr_peaks,
 
 def generate_model(spec):
     """
-    Method that generates a model to fit the hppc data to for peak extraction, using spec dictionary
+    Method that generates a model to fit the RPT data to for peak extraction, using spec dictionary
```
Comment (chirranjeevigopal-TRI) [inline beep/features/featurizer_helpers.py:94]: Update docstring to fix typo: "[...] fit the RPT data for peak extraction [...]"

File: beep/features/featurizer_helpers.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 from scipy.optimize import curve_fit
 
 
-def isolate_dQdV_peaks(processed_cycler_run, diag_nr, charge_y_n, max_nr_peaks, rpt_type, half_peak_width=0.075):
+def isolate_dQdV_peaks(processed_cycler_run, diag_nr, charge_y_n, cwt_range, max_nr_peaks, rpt_type, half_peak_width=0.075):
```
Comment (chirranjeevigopal-TRI) [inline beep/features/featurizer_helpers.py:19]: What does 'cwt' stand for? Can we make the parameter name `cwt_range`  a little more explanatory?

File: beep/features/featurizer_helpers.py [hunk 3]
```diff
@@ -25,7 +25,7 @@ def isolate_dQdV_peaks(processed_cycler_run, diag_nr, charge_y_n, max_nr_peaks,
         rpt_type: string indicating which rpt to pick
         charge_y_n: if 1 (default), takes charge dQdV, if 0, takes discharge dQdV
         diag_nr: if 1 (default), takes dQdV of 1st RPT past the initial diagnostic
-
+        cwt_range: parameter for peakfinder
```
Comment (chirranjeevigopal-TRI) [inline beep/features/featurizer_helpers.py:28]: Does `cwt_range` constrain the range in which to search for peaks?

File: beep/features/featurizer_helpers.py (hunk 2 above)
Comment (BruisVanVlijmen2020) [reply beep/features/featurizer_helpers.py:19]: It is the range of the scaling parameter for the continuous wavelet transform (CWT) algorithm that is used in the peak finding function.

Comment (chirranjeevigopal-TRI) [reply beep/features/featurizer_helpers.py:19]: Got it, if you can mention it in the docstring that would be great

**Pull request merged**